sklearn/tests/test_impute.py

```diff
@@ -464,6 +493,11 @@ def test_imputation_copy():
     Xt[0, 0] = -1
     assert_false(np.all(X == Xt))
 
+    imputer = SamplingImputer(missing_values=0, copy=True)
```

Comment:
Can we use parameterize to avoid duplicate code? The problem with duplication in tests is that it makes it much harder to read what is common between repetitions.

Reply:
Sure, done